### ISSUE TITLE
The nlab's HTML is wonky

### ISSUE THREAD

**Issue by ncfavier** [ncatlab/nlab#29]
...by which I mean [invalid](https://validator.w3.org/check?uri=https%3A%2F%2Fncatlab.org%2Fnlab%2Fshow%2Fcategory&charset=%28detect+automatically%29&doctype=Inline&group=0). While most browsers are able to handle it fine, it would be nice to at least *try* to be compliant.

One issue that stood out in particular is the presence of an entire `<html>` document inside a `<div>`. I'm guessing this is because we generate a full HTML page from each article's contents, and then embed it inside a "template" page with headers and footers.

**Comment by sattlerc** [comment]
Yes, it is really wonky. The page renderer is [implenented](https://github.com/ncatlab/nlab/tree/master/script/src/renderer) in Python and not really maintained. We would like to move to a more standard Markdown to HTML pipeline, but don't have a good understanding of the current page syntax.
